### PR TITLE
Implement /solutions/ai-infrastructure webpage

### PR DESCRIPTION
**Pull request by britneywwc**
## Done

- Implemented `/solutions/ai-infrastructure` webpage as per Figma file and copy docs
- Added forms for `/solutions/ai-infrastructure`, form ID: 5850
- [Figma file](https://www.figma.com/design/youmOu8ZJVBRqeWcC9FYL4/24.10-Canonical.com%2Fsolutions%2Fai-infrastructure?node-id=1-1002&t=VBABcEmgFqniSzGd-0)
- [Copy doc](https://docs.google.com/document/d/1oYV0LsLNg2yUn6Y81AMMX-9jDnojUT8H8kLqqizOZ-M/edit)

## QA
- Go to [demos](https://canonical-com-1266.demos.haus/solutions/ai-infrastructure)
  - Check that design matches Figma and page info matches copy docs
- Submit forms on `Contact Canonical` button and at the bottom of the page: `Send us a message to discuss your AI infrastructure needs`
  - Success banner should popup, redirecting back to `/solutions/ai-infrastructure`

## Issue / Card

Fixes [WD-10497](https://warthogs.atlassian.net/jira/software/c/projects/WD/boards/932?assignee=712020%3A55efc110-e0f6-401a-bcb0-b48a794b6dae&selectedIssue=WD-10497)

## Screenshots

[if relevant, include a screenshot]

[WD-10497]: https://warthogs.atlassian.net/browse/WD-10497?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: templates/solutions/ai-infrastructure/_form-ai-infrastructure.html [hunk 1]
```diff
@@ -0,0 +1,167 @@
...
+        <div class="row--50-50 p-strip is-shallow">
+          <div class="col">
+            <h2 class="p-heading--1">Contact Canonical</h2>
+            <p>Fill in this form and a member of our team will be in touch shortly.</p>
+          </div>
+          <div class="col">
+            <button class="p-modal__close p-modal-close-button "
+                    aria-controls="ai-infrastructure-modal"
+                    aria-label="Close active modal"
+                    style="margin-top: 0">Close</button>
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/_form-ai-infrastructure.html:14]: We actually vanilla classes for [margin](https://vanillaframework.io/docs/utilities/margin-collapse) and padding collapse so no need for the inline style here
```suggestion
            <button class="p-modal__close p-modal-close-button u-no-margin--top"
                    aria-controls="ai-infrastructure-modal"
                    aria-label="Close active modal">Close</button>
```

File: templates/solutions/ai-infrastructure/_form-ai-infrastructure.html [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+                <label for="email" class="is-required">Email:</label>
+                <input required
+                       id="email"
+                       name="email"
+                       maxlength="255"
+                       type="email"
+                       pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </div>
+              <div class="p-section--shallow">
+                <label for="country">Country:</label>
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/_form-ai-infrastructure.html:36]: I think we should use the existing partial for this field, example [here](https://github.com/canonical/canonical.com/blob/f8ec590bdaa74681621c82fce2023ddb95c114a3/templates/solutions/secure-open-source/_form-secure-open-source.html#L26)

File: templates/solutions/ai-infrastructure/index.html [hunk 3]
```diff
@@ -0,0 +1,454 @@
...
+  <section class="p-section">
+    <div class="p-section--shallow">
+      <div class="row--50-50">
+        <hr class="p-rule" />
+        <div class="col">
+          <h2>Deploy on any platform, scale with a hybrid cloud strategy</h2>
+        </div>
+        <div class="col">
+          <p>
+            Choose the ideal AI infrastructure for your use cases – for instance, start quickly with no risk and low investment on public clouds, then move workloads to your own data centre as you scale.
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/index.html:68]: We usually wrap dashes that have spaces on either side
```suggestion
            Choose the ideal AI infrastructure for your use cases &mdash; for instance, start quickly with no risk and low investment on public clouds, then move workloads to your own data centre as you scale.
```

File: templates/solutions/ai-infrastructure/index.html [hunk 4]
```diff
@@ -0,0 +1,454 @@
...
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <h2 class="p-text--small-caps">What customers say</h2>
+    </div>
+    <div class="row">
+      <hr class="p-rule" />
+      <div class="col-3 col-medium-2 p-image-wrapper p-section">
+        {{
+        image (
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/index.html:128]: I've gotten this feedback from Akbar recently so just passing it along, we need to delete the space between `image` and the parenthesis and rerun the djlint for this to be considered correctly formatted. Should look like this 
image

File: templates/solutions/ai-infrastructure/index.html [hunk 5]
```diff
@@ -0,0 +1,454 @@
...
+            <div class="p-logo-section__items">
+              <div class="p-logo-section__item" style="margin-right: 2rem;">
+                {{
+                image (
+                url="https://assets.ubuntu.com/v1/55c9da63-aws-logo.png",
+                alt="AWS",
+                width="50",
+                height="104",
+                hi_def=True,
+                loading="auto|lazy",
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/index.html:195]:
```suggestion
                loading="lazy",
```
These should be either auto or lazy, usually lazy unless they're visible in the vp on load

File: templates/solutions/ai-infrastructure/index.html [hunk 6]
```diff
@@ -0,0 +1,454 @@
...
+      <div class="col">
+        <h2>Security across all layers of the stack</h2>
+      </div>
+      <div class="col">
+        <p class="p-section--shallow">
+          Canonical maintains and supports all the open source tooling in your AI infrastructure stack, including Kubernetes and open source cloud management solutions like OpenStack. Fast-track compliance and run AI projects securely with critical CVEs fixed in under 24h on average.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="">Explore secure open source&nbsp;&rsaquo;</a>
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/index.html:363]: Left a comment on the doc for the missing link

File: templates/solutions/ai-infrastructure/index.html [hunk 7]
```diff
@@ -0,0 +1,454 @@
...
+    </div>
+  </section>
+
+  <div class="u-fixed-width">
+    <hr class="p-rule" />
+  </div>
+  <section class="p-strip is-deep">
+    <div class="u-fixed-width">
+      <h2>
+        <a href="https://canonical.com/solutions/infrastructure">Build your cloud with Canonical&nbsp;&rsaquo;</a>
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/index.html:397]: No need for the full path for local links
```suggestion
        <a href="/solutions/infrastructure">Build your cloud with Canonical&nbsp;&rsaquo;</a>
```

**Review by mtruj013 — CHANGES_REQUESTED** [review]
Thanks @britneywwc, just a few minor fixes needed. As discussed, please take a look at the headings ensuring there are no text orphans as per the figma eg: 

image
vs:
image

**Comment by britneywwc** [comment]
Thanks for the detailed review @mtruj013, I've address your comments in the recent commits. Could you please give it another look?

File: templates/solutions/ai-infrastructure/index.html [hunk 8]
```diff
@@ -0,0 +1,459 @@
...
+  </section>
+
+  <section class="p-section">
+    <div class="u-fixed-width">
+      <h2 class="p-text--small-caps">What customers say</h2>
+    </div>
+    <div class="row">
+      <hr class="p-rule" />
+      <div class="col-3 col-medium-2 p-image-wrapper p-section">
+        {{
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/index.html:135]: Could you apply the same formatting fix here as well?

File: templates/solutions/ai-infrastructure/index.html [hunk 9]
```diff
@@ -0,0 +1,459 @@
...
+          Get peace of mind with Ubuntu certified hardware from all major OEMs. Choosing validated platforms significantly reduces deployment time and costs for end-customers, in addition to improving reliability and providing bug-fixes faster.
+        </p>
+        <p class="p-section--shallow">
+          Combined with NVIDIA DGX systems or NVIDIA-Certified systems, choosing an Ubuntu-certified platform provides secure, performant hardware that is guaranteed to deploy quickly and run smoothly.
+        </p>
+        <hr class="is-muted" />
+        <p>
+          <a href="/partners/silicon">Explore our silicon programme&nbsp;&rsaquo;</a>
+        </p>
+        {{
```
Comment (mtruj013) [inline templates/solutions/ai-infrastructure/index.html:264]: And here

**Review by mtruj013 — APPROVED** [review]
LGTM! Incredibly minor, but looks like these still need reformatting

File: templates/solutions/ai-infrastructure/index.html (hunk 8 above)
Comment (britneywwc) [reply templates/solutions/ai-infrastructure/index.html]: Thanks for the catch!

**Pull request merged**